.github/workflows/my-ci-pipeline.yml

```diff
@@ -12,5 +12,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - run: docker build --target test --tag todo-app:test .
-    - run: docker run --env-file .env.test todo-app:test
\ No newline at end of file
+    - run: docker build --target test --tag todo-app:test --cache-from jormondroyd/todo-app .
```

Comment:
I can't remember if we said elsewhere, but it looks like this cache-from isn't working - I can't see any layers being cached. You may need to explicitly pull the existing version before this, e.g. add a new step with `docker pull jormondroyd/todo-app`